src/hotspot/share/cds/aotConstantPoolResolver.cpp

```diff
@@ -0,0 +1,573 @@
+/*
+ * Copyright (c) 2022, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If this is a new file, it shouldn't have 2022 in the copyright year.

Reply:
Oh looks like it's been moved from classPrelinker, except git doesn't show it's moved.  If it's added as a moved file, would there be less diffs to look at?